cilium-cli/cli/connectivity.go

```diff
@@ -58,6 +58,10 @@ func RunE(hooks api.Hooks) func(cmd *cobra.Command, args []string) error {
 		params.ImpersonateAs = impersonateAs
 		params.ImpersonateGroups = impersonateGroups
 
+		// Remove trailing dot from external-target paramater to make it compatible for
+		// use as an SNI
+		params.ExternalTarget, _ = strings.CutSuffix(params.ExternalTarget, ".")
```

Comment:
nit: `strings.TrimSuffix` is the more appropriate helper here

Reply:
Thanks, fixed!